src/scripts/main.js

```diff
@@ -354,7 +354,26 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+function createTableRow(tableId) {
+  people.age = people.born - people.died;
+  people.century = Math.ceil(people.died / 100);
 
-// write your code here
+  const table = document.querySelector('dashboard');
+
+  const properties = ['name', 'age', 'city', 'job', 'phone', 'email'];
```

Comment:
The 'properties' array includes 'city', 'job', 'phone', and 'email', which do not exist in the 'people' objects. You should only include properties that exist, such as 'name', 'age', and 'century'.